gb-cpu/src/processor/cpu/registers/area.rs

```diff
@@ -0,0 +1,41 @@
+#[derive(Debug)]
+pub enum Bits8 {
+    B,
+    C,
+    D,
+    E,
+    H,
+    L,
+}
+
+#[derive(Debug)]
+pub enum Bits16 {
```

Comment:
Where's `AF` ?

Reply:
AF shouldn't be a valid register. A is the accumulator and F the flags, they are never meant to be manipulated as a 16 bit register, that's why i don't include it here

Reply:
these are valid register that are decoded from the opcode, you may re-read [Gb Cpu manual](http://marc.rawer.de/Gameboy/Docs/GBCPUman.pdf) at the page 61

Reply:
Great catch, i'll put it in then!